homeassistant/components/blebox/manifest.json

```
@@ -0,0 +1,9 @@
+{
+  "domain": "blebox",
+  "name": "BleBox devices",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/blebox",
+  "requirements": ["blebox_uniapi==1.3.0", "semver==2.9.1"],
```

```suggestion
  "requirements": ["blebox_uniapi==1.3.0"],
```
Doesn't look like semver is used

---

Thanks!